examples/hello-spring-boot/.gitignore

```diff
@@ -0,0 +1,8 @@
+target
+.classpath
+.project
+.settings
+.vscode
+.idea/*
+!.idea/
```

Comment:
Do you need ‘!.idea’ given that your are excluding the xml file itself?

Reply:
let me check, I was wrestling with these rules a bit and might have over-complicated.

Reply:
You're correct, not necessary! Removed it, thanks!